.github/docker/Dockerfile.node

```diff
@@ -1,3 +1,3 @@
-FROM node:20.14.0-alpine3.20
+FROM node:20.14.0-bookworm
```

Comment:
Switched to build images using bookworm just so we have the same version of everything (distroless is Debian 12 bookworm).